tools/tpm2_getmanufec.c

```diff
@@ -447,7 +447,7 @@ int RetrieveEndorsementCredentials(char *b64h)
         goto out_easy_cleanup;
     }
 
-    rc = curl_easy_setopt(curl, CURLOPT_VERBOSE, respfile);
+    rc = curl_easy_setopt(curl, CURLOPT_VERBOSE, respfile == NULL ? 0L : 1L);
     if (rc != CURLE_OK) {
```

Comment:
You don't need to use an unary opeator here, just use respfile != NULL as @williamcroberts mentioned.